tests/unit/amo/components/TestAutoSearchInput.js

```diff
@@ -297,6 +297,28 @@ describe(__filename, () => {
       );
     });
 
+    it('can be used without a location', () => {
+      const { store } = dispatchClientMetadata();
+      const dispatch = sinon.spy(store, 'dispatch');
+      const root = render({
+        store,
+      });
+
+      const query = 'ad blocker';
+      fetchSuggestions({ root, query });
+
+      sinon.assert.calledWith(
+        dispatch,
+        autocompleteStart({
+          errorHandlerId: root.instance().props.errorHandler.id,
```

Comment:
It would be better to use `createStubErrorHandler()` and pass it to `render()`.

Reply:
Yes, I really just copied an existing test and only made the changes necessary to it.